sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/MetricFeedback/MetricFeedback.cs

```diff
@@ -170,6 +170,12 @@ internal static MetricFeedback DeserializeMetricFeedback(JsonElement element)
             return new UnknownMetricFeedback(feedbackType, feedbackId.Value, Optional.ToNullable(createdTime), userPrincipal.Value, metricId, dimensionFilter);
         }
 
+        internal static MetricFeedback FromResponse(Response response)
+        {
+            using var document = JsonDocument.Parse(response.Content);
```

Comment:
What happens if the response.Content is empty and cannot be parsed?

Reply:
If an empty response.Content is passed, then `JsonDocument.Parse(new BinaryData(Array.Empty<byte>()))` throws `JsonReaderException`

![image](https://user-images.githubusercontent.com/45216704/162088628-8cf1b7f0-dd04-435e-8b7f-20c85a629787.png)

Reply:
Since the caller doesn't pass the RequestContext here, we don't need to add null check here.